VisualQnA/docker_compose/intel/hpu/gaudi/compose.yaml

```diff
@@ -34,6 +34,7 @@ services:
       - llava-tgi-service
     ports:
       - "9399:9399"
+    runtime: runc
```

Comment:
I think we don’t need to specify runtime `runc` manually, as Docker uses `runc` by default. Why do we need to set it?